src/pages/Categories/Categories.tsx

```diff
@@ -15,20 +17,31 @@ import PreLoader from "@components/PreLoader/PreLoader";
 
 
 const Categories: FC = () => {
+    const [groups, setGroups] = useState<IGroup[]>([]); 
```

Comment:
We save second time in our localStorage items, this kills the RTK Query logic as an automation cache